elephant/current_source_density.py

```diff
@@ -253,10 +253,31 @@ def generate_lfp(csd_profile, x_positions, y_positions=None, z_positions=None,
 
     Returns
     -------
-    LFP : neo.AnalogSignal
+    LFP : :class:`neo.core.AnalogSignal`
```

Comment:
This change should also apply in line 74, right? 
https://github.com/NeuralEnsemble/elephant/blob/ade17e8b885018dddd0ea2983ef275a83123cad0/elephant/current_source_density.py#L74
(Can't directly suggest the change here, because it's too far from any lines changed in this PR.